### ISSUE TITLE
problem when trying to install

### ISSUE THREAD

**Issue by posseidon23** [coin-or/COIN-OR-OptimizationSuite#1]
at the "make" step, I get the error message :
"./../../FlopCpp/src/MP_set.hpp:220:22: error: no matching constructor for
      initialization of 'flopc::MP_domain'
          return MP_domain(s);}" 

does anybody know how to fix?

**Comment by tkralphs** [comment]
From the error, I'm guessing you're building on OS X. `FlopCpp` doesn't build on OS X with clang and I haven't been able to figure out a fix. Unless you are specifically interested in using `FlopCpp`, the fix is just to skip building it by configuring with the argument `COIN_SKIP_PROJECTS=FlopCpp`. Let me know if this fixes the problem. 

In the next couple of days, I'll try to update the README with better instructions on building. The most detailed and up-to-date instructions are in the slides for a course I recently gave on Computational MILP, starting around slide 16 here:

http://coral.ie.lehigh.edu/~ted/files/computational-mip/lectures/Lecture3.pdf

The instructions for SYMPHONY [here](https://projects.coin-or.org/SYMPHONY/wiki/BuildAndInstall) can also be modified for the Optimization Suite.

**Comment by posseidon23** [comment]
Thank you professor,
yes I am building from OS X.  
I tried your suggestion, but had this at the end of 'make test' : 
"....... 
128 warnings generated.
make[4]: *** No rule to make target `/Coin-Or/build/Couenne/src/main/libBonCouenne.la', needed by `OSSolverService'.  Stop.
make[3]: *** [all-recursive] Error 1
make[2]: *** [all] Error 2
make[1]: *** [all-recursive] Error 1
make: *** [all-recursive] Error 1
"

**Comment by tkralphs** [comment]
Hmm, this error looks familiar, but it seems to me that I fixed this problem. What version of the Optimization Suite are you building? And how did you get the source? Also, what is the compiler and compiler version and what OS X version? Actually, if you could just give the full `configure` command that you used, that would be helpful to replicate. I'm also going to set up a Travis build, which I should have done already.

**Comment by posseidon23** [comment]
Hi professor Ralph,
thanks for answering. ok, the version I am trying to install is coin Optimization suite 1.7. I downloaded it with subversion. Actually, when I try to install the components individually, it works, at least cbc, clp, symphony(the one I need most), but I wanted something that links them, that is why I am trying to install the optimization suite. For the configure command is : (I created the directory build, enter build and run the following command):
_**../configure COIN_SKIP_PROJECTS=FlopCpp.**_
Well I am not proficient in shell command, so whenever I find instruction, I try to run it as is described .
Maybe the version matters, as I started to build the 1.7 version I realized there was a more recent one (1.8), I downloaded this latter, but did it not work, the message I have is : 
 **error: source directory already configured; run "make distclean" there first** 
(which I did), but maybe I have to remove it.
